lark/parsers/lalr_parser.py

```diff
@@ -132,7 +133,7 @@ def feed_token(self, token, is_end=False):
 
             if action is Shift:
                 # shift once and return
-                assert not is_end
+                # assert not is_end
```

Comment:
Wonder what would happen if you undid this

Reply:
It's just an assert. It doesn't matter.